src/features/callAssignments/components/CallAssignmentCallersList.tsx

```diff
@@ -14,25 +13,7 @@ import { Msg, useMessages } from 'core/i18n';
 import messageIds from '../l10n/messageIds';
 import theme from '../../../theme';
```

Comment:
Same here with removing import of `theme`, and using it from the function passed to `sx` !